Acrolinx.Sidebar.Tests/AcrolinxStorageTest.cs

```diff
@@ -29,43 +28,22 @@ public void DefaultStorage()
             value = storage.GetItem("acro.test.token2");
 
             Assert.AreEqual(value, "tokendata2");
-
-            string defaultPath = Environment.GetFolderPath(Environment.SpecialFolder.UserProfile) + @"\.acrolinx\" + Util.AssemblyUtil.AppInfo()["applicationName"] + "_localStorage.json";
-
-            Assert.AreEqual(File.Exists(defaultPath), true);
-
-            File.Delete(defaultPath);
         }
 
         [TestMethod]
-        public void SetItemGetItemInitStorage()
+        public void GetItemFromHKLM()
         {
-            JSONAcrolinxStorage storage = JSONAcrolinxStorage.Instance;
-
-            storage.InitStorage(Path.GetTempPath() + @"test_dotnet_sdk\test.json");
-
-            storage.SetItem("acro.test.token1", "tokendata1");
-            storage.SetItem("acro.test.token2", "tokendata2");
+            RegistryAcrolinxStorage storage = RegistryAcrolinxStorage.Instance;
 
-            var value = storage.GetItem("acro.test.token1");
-
-            Assert.AreEqual(value, "tokendata1");
-
-            value = storage.GetItem("acro.test.token2");
+            var value = storage.GetItem("unknown");
 
-            Assert.AreEqual(value, "tokendata2");
-
-            Assert.AreEqual(File.Exists(Path.GetTempPath() + "/test_dotnet_sdk/test.json"), true);
-
-            File.Delete(Path.GetTempPath() + "/test_dotnet_sdk/test.json");
+            Assert.AreEqual(value, null);
         }
 
         [TestMethod]
         public void RemoveItem()
         {
-            JSONAcrolinxStorage storage = JSONAcrolinxStorage.Instance;
-
-            storage.InitStorage(Path.GetTempPath() + "/test_dotnet_sdk/test.json");
+            RegistryAcrolinxStorage storage = RegistryAcrolinxStorage.Instance;
 
             storage.SetItem("acro.test.token1", "tokendata1");
             storage.SetItem("acro.test.token2", "tokendata2");
```

Comment:
I would at least overwrite it once otherwise it could be that you only test the read and not the write or overwrite fails at all

Reply:
Added UT for overwrite